blocks/library/cover-image/style.scss

```diff
@@ -32,6 +32,50 @@
 		background: rgba( 0,0,0,.5 );
 	}
 
+	&.has-background-dim.has-background-dim--00:before {
+		background: rgba( 0,0,0,0 );
+	}
+
+	&.has-background-dim.has-background-dim--01:before {
+		background: rgba( 0,0,0,.1 );
+	}
+
+	&.has-background-dim.has-background-dim--02:before {
+		background: rgba( 0,0,0,.2 );
+	}
+
+	&.has-background-dim.has-background-dim--03:before {
+		background: rgba( 0,0,0,.3 );
+	}
+
+	&.has-background-dim.has-background-dim--04:before {
+		background: rgba( 0,0,0,.4 );
+	}
+
+	&.has-background-dim.has-background-dim--05:before {
+		background: rgba( 0,0,0,.5 );
+	}
+
+	&.has-background-dim.has-background-dim--06:before {
+		background: rgba( 0,0,0,.6 );
+	}
+
+	&.has-background-dim.has-background-dim--07:before {
+		background: rgba( 0,0,0,.7 );
+	}
+
+	&.has-background-dim.has-background-dim--08:before {
+		background: rgba( 0,0,0,.8 );
+	}
+
+	&.has-background-dim.has-background-dim--09:before {
+		background: rgba( 0,0,0,.9 );
+	}
+
+	&.has-background-dim.has-background-dim--10:before {
```

Comment:
A SASS loop control could help avoid the duplication here, if maybe a bit overkill:

http://sass-lang.com/documentation/file.SASS_REFERENCE.html#_for
http://thesassway.com/intermediate/if-for-each-while#for

```sass
@for $i from 1 through 10 {
	&.has-background-dim.has-background-dim-#{ $i } {
		background-image: rgba( black, $i * 0.1 );
	}
}
```

(Which, I'm seeing could be an issue for the 0 prefixing, and begs the question: Do we need the padding?)

Reply:
Yeah, I'll drop the padding. I think my subconscious reasoning was that classes are strings, thus numbers injected into these should have the padding so that alphabetical sorting matches number sorting. ¯\_(ツ)_/¯